src/bsoncxx/document/element.cpp

```diff
@@ -44,31 +44,21 @@ namespace bsoncxx {
 BSONCXX_INLINE_NAMESPACE_BEGIN
 namespace document {
 
-element::element() : _raw(nullptr), _length(0), _offset(0) {}
+element::element() : element(nullptr, 0, 0) {}
```

Comment:
Noticed while doing constructor changes - no need to re-iterate the field names, just delegate.